### DISCUSSION TITLE
Menu items not localized as expected

### DISCUSSION THREAD

**Discussion by duncan-fairley** [Shopify/storefront-api-feedback#180] [Help] [answered]
Hello!

Very excited to see Shopify make more items localizable. 

We're encountering a problem where the following query is returning the english version of Online Store menus despite being translated through Translate and Adapt. 
```graphql
query @inContext(language: FR) {
  menu(handle: "header") {
    items {
        title
    }
  }
}
```

Response (request ID: `769aeecd-5516-4c76-83b6-c44c996f88fc`)
```json
{
   "data": {
      "menu": {
         "items": [
            {
               "title": "Paddle Boards"
            },
            {
               "title": "Surf Boards"
            },
            {
               "title": "Gear"
            },
            {
               "title": "Explore"
            },
            {
               "title": "Closeouts"
            }
         ]
      }
   },
   "extensions": {
      "context": {
         "country": "US",
         "language": "FR"
      }
   }
}
```
<img width="1321" alt="Screenshot 2023-03-20 at 8 50 58 PM" src="https://user-images.githubusercontent.com/995071/226513499-3429dd6d-3791-47a4-bcef-adb69baaf478.png">

**Comment by frehner** [comment, marked as answer]
Hello @DuncanFa , there is an existing discussion open here for this. Thanks!

https://github.com/Shopify/storefront-api-feedback/discussions/156

**Reply by frehner** [reply]
(Make sure to "upvote" that issue so it gets more visibility)
